thrift/thrift0.13/src/main/java/com/linecorp/armeria/server/thrift/ThriftCallServiceBuilder.java

```diff
@@ -0,0 +1,81 @@
+/*
+ * Copyright 2024 LINE Corporation
+ *
+ * LINE Corporation licenses this file to you under the Apache License,
+ * version 2.0 (the "License"); you may not use this file except in compliance
+ * with the License. You may obtain a copy of the License at:
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+ * WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+ * License for the specific language governing permissions and limitations
+ * under the License.
+ */
+
+package com.linecorp.armeria.server.thrift;
+
+import static com.google.common.collect.ImmutableMap.toImmutableMap;
+import static java.util.Objects.requireNonNull;
+
+import java.util.Map;
+import java.util.concurrent.Executors;
+
+import com.google.common.collect.ImmutableList;
+import com.google.common.collect.ImmutableMap;
+
+/**
+ * A fluent builder to build an instance of {@link ThriftCallService}.
+ *
+ * @see ThriftCallService
+ */
+public final class ThriftCallServiceBuilder {
+    private Map<String, ? extends Iterable<?>> implementations;
```

Comment:
What do you think about making this additive, like:

```java
private final ImmutableMap.Builder<String, ...> implementations = ImmutableMap.builder();
...
public ThriftCallService build() {
    ...
    final Map<...> implementations = this.implementations.build();
    ...
}

// User code:
ThriftCallService
  .builder()
  .addService(defaultServceImpl)
  .addService("foo", fooServiceImpl)
  .addService("bar", barServiceImpl)
  .addServices(ImmutableMap.of("qux", quxServiceImpl, "quz", quzServiceImpl))
  .build();

// implementations will contain "", "foo", "bar", "qux", "quz".
```

Reply:
It looks good.
I applied it. Thank you for your comment.